Userland/Libraries/LibIPC/Connection.h

```diff
@@ -101,6 +101,10 @@ class Connection : public Core::Object {
             total_nwritten += nwritten;
         }
 
+#ifdef __serenity__
+        (void)donate_peer(m_socket->fd());
```

Comment:
nitpick: should this be a separate commit, it's not mentioned in the title / description.